README.md

```diff
@@ -184,3 +184,31 @@ class Interpolate a where
 
 data HasClass c = forall a. c a => HasClass a
 ```
+
+### `implicit-no-builder`
```

Comment:
Can you update the examples code block?

Reply:
done.

Reply:
Sorry, I wasn't very clear. I meant this code block: https://github.com/brandonchinn178/string-syntax/blob/d5d4bf0a7d13edceb361e71f3a2a989c5c7188dd/README.md?plain=1#L7-L14